.github/workflows/ci.yml

```diff
@@ -13,12 +13,12 @@ jobs:
       - run: composer install --prefer-dist --no-interaction
       - run: bin/php-cs-fixer fix --ansi --dry-run --using-cache=no --verbose
 
-  tests:
-    name: Tests
+  atoum-tests:
+    name: Atoum tests
```

Comment:
much discrimination, such sadness

Reply:
I'm planing to add a bit of rector & phpstan, that's why :grin:

Reply:
=> #89